IOS/SwiftSeedProject/SwiftSeedProject/Infrastructure/Network/RestClient.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Api.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 4/4/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+import SwiftyJSON
+
+final class RestClient {
+    // MARK: - Private Properties
+    private let manager: ApiManagerProtocol
+    private let persistence: Persistence
+    private let processSources: (Persistence, Result<JSON>, (ApiResult<Array<Source>>) -> Void) -> Void = { persistence, result, completion in
```

Comment:
Yep, but to move on by now I would add a TODO comment, and then refactor this (there are a few thing that we need to improve on RestClient)